packages/core/upload/admin/src/components/AssetDialog/tests/DialogHeader.test.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import { IntlProvider } from 'react-intl';
+import { QueryClientProvider, QueryClient } from 'react-query';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { useFolderStructure } from '../../../hooks/useFolderStructure';
+import { DialogHeader } from '../DialogHeader';
+
+jest.mock('../../../utils', () => ({
+  ...jest.requireActual('../../../utils'),
+  getTrad: x => x,
+}));
+
+jest.mock('../../../hooks/useFolderStructure');
+
+const setup = props => {
+  const withDefaults = {
+    currentFolder: null,
+    onChangeFolder: jest.fn(),
+    ...props,
+  };
+
+  const queryClient = new QueryClient({
+    defaultOptions: {
+      queries: {
+        retry: false,
+        refetchOnWindowFocus: false,
+      },
+    },
+  });
+
+  return render(
+    <QueryClientProvider client={queryClient}>
+      <ThemeProvider theme={lightTheme}>
+        <IntlProvider locale="en" messages={{}}>
+          <DialogHeader {...withDefaults} />
+        </IntlProvider>
+      </ThemeProvider>
+    </QueryClientProvider>
+  );
+};
+
+describe('Upload || components || DialogHeader', () => {
+  it('should render folder name and back button', () => {
+    useFolderStructure.mockReturnValueOnce({
+      isLoading: false,
+      error: null,
+      data: [
+        {
+          value: null,
+          label: 'Media Library',
+          children: [
+            {
+              value: 1,
+              label: 'Cats',
+              children: [
+                {
+                  value: 2,
+                  label: 'Michka',
+                  children: [],
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    });
```

Comment:
I've added a global mock for the hook which should return a structure already, that we can test on. Could you update the test for it? https://github.com/strapi/strapi/blob/9b7c0ca077b296db71f221d0850fcf2c4ea872ef/packages/core/upload/admin/src/hooks/__mocks__/useFolderStructure.js